cmd/titus-log-uploader/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/Netflix/metrics-client-go/metrics"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Netflix/titus-executor/filesystems"
+	kubeviewer "github.com/Netflix/titus-executor/logviewer/kubelet"
+	"github.com/Netflix/titus-executor/tag"
+	"github.com/Netflix/titus-executor/uploader"
+)
+
+/*
+	This binary handles log persistence and access from within a pod.
+
+	It accomplishes this by having an injected shared volume and a webserver.  The binary monitors this shared volume and publishes to s3 on a specified cadience.  Additionally it provides a simple web server for stating and listing these logs in accordance with the logviewer API from the titus-api server.
```

Comment:
Nit; long line.

Reply:
https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings